src/style.css

```diff
@@ -0,0 +1,78 @@
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.block-section {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block-item {
+  display: flex;
+  font-size: 100px;
+  height: 300px;
+  color: white;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 100%;
+}
+
+.block-1 {
+  background-color: #f00;
+  order: 1;
+}
+
+.block-2 {
+  background-color: #c00;
+  order: 2;
+}
+
+.block-3 {
+  background-color: #900;
+  order: 3;
+}
+
+.block-4 {
+  background-color: #600;
+  order: 4;
+}
+
+.block-5 {
+  background-color: #300;
+  order: 5;
+}
+
+.block-6 {
+  background-color: black;
+  order: 6;
+}
```

Comment:
you don't need to provide orders, if they're the same as order in html

Reply:
I mean, first block already first in html and so on

Reply:
If I delete 5 and 6 it doesn't work